include/linux/soundwire/sdw.h

```diff
@@ -547,6 +547,10 @@ struct sdw_slave_ops {
  * @node: node for bus list
```

Comment:
In commit message, a very minor nit:

The time difference between the end of the enumeration process and the
call to .update_status() **may be** as small as a SoundWire frame, so with a
48kHz frame rate, .update_status() **can** be invoked in 10s of
microseconds.

Did you mean this? I read the original that the difference is always length of a SW frame and unless there are scheduling hick-ups, .update_status() is called. Oh well, a very minor nit.

Reply:
yes, the wording is not quite right

Should be something like

The time difference between the end of the enumeration process and a change of status reported by the hardware may be as small as one SoundWire frame. .update_status() will be invoked in an interrupt thread. With a 48 kHz frame rate and ideal scheduling cases, the time difference at the software level will be double-digit microseconds, short enough to create a race condition.